packages/scaffold-ui/src/partials/w3m-alertbar/styles.ts

```diff
@@ -0,0 +1,14 @@
+import { css } from 'lit'
+
+export default css`
+  :host {
+    display: block;
+    position: absolute;
+    opacity: 1;
```

Comment:
Do we need `opacity` to be set true? Since it's already 1 by default

Reply:
Oh yeah we don't need it 👍